content/docs/tasks/traffic-management/custom-gateway/index.md

```diff
@@ -0,0 +1,385 @@
+---
+title: Add a new Custom Ingress Gateway
```

Comment:
I think "new" is redundant here, custom ingress gateway is enough. Also, I would use "deploy" instead of "add", so the tile could be "Deploy a custom ingress gateway".

Reply:
I also think that an important issue here is the cert-manager. I would consider calling this task: "Deploy a custom ingress gateway using cert-manager"